### ISSUE TITLE
Status does not clear when finished with episode or movie

### ISSUE THREAD

**Issue by kabili207** [oonqt/MBCord#15]
**Describe the bug**
After watching something in Emby, the discord status will not clear

**Expected behavior**
The status should clear when finished watching a movie or episode

**Error**
[MBCord-1608146294.txt](https://github.com/oonqt/MBCord/files/5713453/MBCord-1608146294.txt)
```
[Thu Dec 17 2020 22:05:45 GMT-0500 (Eastern Standard Time)] error: Failed to set activity: TypeError: Cannot read property 'toLowerCase' of undefined
```

**Desktop**
 - OS: Gentoo Linux
 - Server Type: Emby 4.5.2.0
 - Version 2.1.9

**Comment by oonqt** [comment]
I have ran into this error myself quite often, it'll be corrected in the next version which should be coming within the next few weeks, if not earlier.

**Comment by oonqt** [comment]
Fixed in [2.1.10](https://github.com/oonqt/MBCord/releases/tag/2.1.10)
